node/src/main.rs

```diff
@@ -128,6 +130,7 @@ async fn main() {
                 .conflicts_with_all(&["ethereum-ws", "ethereum-ipc"])
                 .long("ethereum-rpc")
                 .value_name("NETWORK_NAME:URL")
+                .env("ETHEREUM_RPC")
```

Comment:
Can we add the sae for `--ethereum-ws` and `--ethereum-ipc` (that's usually local, so perhaps not important).

Reply:
Should probably also add an environment variable for `--ipfs`.